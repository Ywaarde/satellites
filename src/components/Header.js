import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
    <div>
      <h1>Satellites</h1>
      <NavLink to='/' activeClassName='is-active' exact={true}>Misson Overview</NavLink>
      <NavLink to='/create' activeClassName='is-active'>Add mission</NavLink>
    </div>
)

export default Header;