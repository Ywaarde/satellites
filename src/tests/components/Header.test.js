import React from 'react';
import { shallow } from 'enzyme';
// import toJson from 'enzyme-to-json';
import Header from '../../components/Header'

test('should render the header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find('h1').text()).toBe('Satellites');
    // const renderer = new ReactShallowRenderer()
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot()
})

