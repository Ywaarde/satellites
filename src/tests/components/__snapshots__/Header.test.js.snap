// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the header correctly 1`] = `
<div>
  <h1>
    Satellites
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Misson Overview
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Add mission
  </NavLink>
</div>
`;
